### PR TITLE
Multi ticket typo fixes

### PR DESCRIPTION
**Pull request by karbates**
- FHIR-33525: Swapped CDC link on COVID-19 for WHO link in Scope section of index.md
- FHIR-33529: Fixed typo, from "steam" to "stream" on contact.md
- FHIR-33597: Swapped colon for period after "US profiles of Patient" and added "to" after two instances of "SHALL conform" in patient.md
- FHIR-33598: Changed [] to () around VaccineTypeSNOMED so they render as a link in vaccination.md

### CODE REVIEW

**Comment by p9g** [comment]
The fix for FHIR-33598 does not work.  See https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/branches/multi-ticket-typo-fixes/vaccination.html 
This is related to https://github.com/HL7/fhir-shc-vaccination-ig/blob/master/examples/byCodeSystem/README.md that has a column for Value Set which has terms like VaccineTypeSNOMED which are not used in the IG.

File: input/pagecontent/vaccination.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Implementers SHALL use at least one of the following code systems for identifyin
...
-* A **value set** defines a group of codes that can be used within a specific element. For example, we define a [value set of SNOMED-CT codes for vaccine products][VaccineTypeSNOMED] for use in the `vaccineCode` element. FHIR allows an element to be "bound" to a value set, meaning that the contents of that element must come from the value set (for a "[required binding][binding]") or should come form the value set (for a "[preferred binding][binding]").
+* A **value set** defines a group of codes that can be used within a specific element. For example, we define a [value set of SNOMED-CT codes for vaccine products](VaccineTypeSNOMED) for use in the `vaccineCode` element. FHIR allows an element to be "bound" to a value set, meaning that the contents of that element must come from the value set (for a "[required binding][binding]") or should come form the value set (for a "[preferred binding][binding]").
```
Comment (p9g) [inline input/pagecontent/vaccination.md:50]: (VaccineTypeSNOMED) does not work.

Comment (p9g) [reply input/pagecontent/vaccination.md:50]: See https://raw.githubusercontent.com/HL7/fhir-shc-vaccination-ig/dev/input/includes/markdown-link-references.md  perhaps [VaccineSNOMED] would work

**Comment by p9g** [comment]
https://github.com/HL7/fhir-shc-vaccination-ig/pull/192 has applied some of these same changes

**Comment by masnick** [comment]
Covered by #192

**Pull request closed without merging**